dipy/direction/peaks.py

```diff
@@ -552,15 +551,6 @@ def peaks_from_model(model, data, sphere, relative_peak_threshold,
                            odf_array if return_odf else None)
 
 
-def gfa(samples):
-    """The general fractional anisotropy of a function evaluated
```

Comment:
Oh ok, I see. Sorry and thanks  !

Even if it does not have any bad effect, I do not understand why you want to keep this "old" import `from dipy.direction.peaks import gfa` in the aformentioned test?